README.MD

```diff
@@ -1,33 +1,81 @@
 
-### Ubuntu
-Just build the docker by executing:  
+## Server images
+Basic instructions to build and run the TeaSpeak server images yourself.
+
+### Alpine (~183MB, 13 Layers)
+```shell script
+# Build the Image with the server version 1.4.12 (default)
+docker build -f server/alpine.Dockerfile -t teaspeak/server:1.4.12-alpine .
+
+# alternatively with server version x.y.z
+# docker build -f server/alpine.Dockerfile --build-arg TEASPEAK_VERSION=x.y.z -t teaspeak/server:x.y.z-alpine .
+
+# run the Alpine based server image
+docker run -d \
+  -v teaspeak_logs:/ts/logs \
+  -v teaspeak_db:/ts/database \
+  -v teaspeak_files:/ts/files \
+  -v teaspeak_certs:/ts/certs \
+  -p 9987:9987/udp -p 10101:10101/tcp -p 30303:30303 \
+  --restart=unless-stopped --name teaspeak-server teaspeak/server:1.4.12-alpine
 ```
-docker build --tag teaspeak:ubuntu server/ubuntu/
-```  
-Running the docker:  
+
+### Debian (~445MB, 11 Layers)
+```shell script
+# Build the Image with the server version 1.4.12 (default)
+docker build -f server/debian.Dockerfile -t teaspeak/server:1.4.12-debian .
+
+# alternatively with server version x.y.z
+# docker build -f server/debian.Dockerfile --build-arg TEASPEAK_VERSION=x.y.z -t teaspeak/server:x.y.z-debian .
+
+# run the Debian based server image
+docker run -d \
+  -v teaspeak_logs:/ts/logs \
+  -v teaspeak_db:/ts/database \
+  -v teaspeak_files:/ts/files \
+  -v teaspeak_certs:/ts/certs \
+  -p 9987:9987/udp -p 10101:10101/tcp -p 30303:30303 \
+  --restart=unless-stopped --name teaspeak-server teaspeak/server:1.4.12-debian
 ```
-docker run --rm -i teaspeak:ubuntu
-```  
-Run with parameters:
+
+### Ubuntu (~437MB, 14 Layers)
+```shell script
+# Build the Image with the server version 1.4.12 (default)
+docker build -f server/ubuntu.Dockerfile -t teaspeak/server:1.4.12-ubuntu .
+
+# alternatively with server version x.y.z
+# docker build -f server/ubuntu.Dockerfile --build-arg TEASPEAK_VERSION=x.y.z -t teaspeak/server:x.y.z-ubuntu .
+
+# run the Ubuntu based server image
+docker run -d \
+  -v teaspeak_logs:/ts/logs \
+  -v teaspeak_db:/ts/database \
+  -v teaspeak_files:/ts/files \
+  -v teaspeak_certs:/ts/certs \
+  -p 9987:9987/udp -p 10101:10101/tcp -p 30303:30303 \
+  --restart=unless-stopped --name teaspeak-server teaspeak/server:1.4.12-ubuntu
 ```
-docker run --rm -i teaspeak:ubuntu -Pgeneral.database.url=sqlite://database/TeaData.sqlite
+
+## Logs and Token
+To view the docker container logs and to find the server token you have to run the following command:
+```shell script
+docker logs <container-name|container-id>
+
+# to get the container name or id run the following
+docker ps 
+
+# eg. if the server gets started as displayed in the above section
+docker logs teaspeak-server
+
+# and if you want to follow the logs (ctrl+c to stop following)
+docker logs -f teaspeak-server
 ```
 
+## Web Image
+All references to build the TeaSpeak Web client in a Docker container
```

Comment:
Build or just spin up a web server?
Note: if you want to know how to build the web client lookup this file: https://github.com/TeaSpeak/TeaWeb/blob/master/scripts/travis.sh

I've not yet 100% updated the setup guide since I recently switched to WebPack and adopted React.

Reply:
Nice, thy for this info!
TBH I wanted to try and do both, given that your existing file seemed to be a builder image, which allowed the user to build the web client (or you in your build pipelines) and a simple Image which allowed the user or you to host the web client in a container.
(yes I know my readme text doesn't reflect that yet :thinking: see it as WIP :wink: )

Reply:
Okay, I scrapped the idea of a builder image for now based on our discussion in the official ts server